.github/workflows/run-tests.yml

```diff
@@ -25,9 +25,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install coverage
+        python -m pip install coverage flake8
         python -m pip install -r requirements.txt
 
-    - name: Run tests with Coverage
-      run: |
-        cd test && nosetests --with-coverage
+    - name: Enforce Github editor 127 character width
+      run: flake8 ./pybsn/ ./bin/* --count --max-complexity=20 --max-line-length=127 --statistics
```

Comment:
since the python files in `bin` do not end in `.py`, use shell expansion to pass them explicitly to be checked by flake 8 (by default, it will only check `.py` files in directories).